src/routes/blog/content.ts

```diff
@@ -51,6 +52,7 @@ export const posts = Object.entries(postsGlob)
             title: frontmatter.title,
             description: frontmatter.description,
             date: new Date(frontmatter.date),
+            lastUpdated: new Date(frontmatter.lastUpdated),
```

Comment:
Anyway we can get this from git history rather than frontmatter?

Reply:
That sounds like a better approach, I'll find and see what we can do.

Reply:
@TorstenDittmann do you think we could do this?
Afaik, this could mark any minor typo or short fixes as an article update, wdyt?

Reply:
lets not do it via git history, I think this approach is fine until we got a cms 👍🏻